docs/data-sources/user.md

```diff
@@ -0,0 +1,33 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "materialize_user Data Source - terraform-provider-materialize"
+subcategory: ""
+description: |-
+  The user data source allows you to retrieve information about a user in your Frontegg organization.
```

Comment:
```suggestion
  The user data source allows you to retrieve information about a user in your Materialize organization.
```

Reply:
FrontEgg is an implementation detail; users shouldn't need to know that this is the authentication provider we use.